cfme/tests/containers/test_ad_hoc_matrices.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from utils import testgen
+from utils.log import logger
+from cfme.containers.provider import ContainersProvider
+from utils.version import current_version
+from utils.appliance.implementations.ui import navigate_to
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.8"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
+
+
+@pytest.fixture
+def metrics_up_and_running(provider):
+    import requests
```

Comment:
Don't worry importing requests at the top of the file, normally we only import inside functions when the top-level import would cause a circular import.